### ISSUE TITLE
SVC(coef0)란?

### ISSUE THREAD

**Issue by GzuPark** [machinelearning-pangyo/Hands-On-MachineLearning#3]
http://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html

설명:
Independent term in kernel function. It is only significant in ‘poly’ and ‘sigmoid’.
